### ISSUE TITLE
Bento Clone of amp-social-share: Image-based buttons with missing text descriptions / accessible name

### ISSUE THREAD

**Issue by krdwan** [ampproject/amphtml#30738]
Cloned of https://github.com/ampproject/amphtml/issues/29563
For Bento

Opening a clone bug for Bento.  Original bug addressed in `0.1` version of component.
